storage/innobase/log/log0recv.cc

```diff
@@ -1232,6 +1231,18 @@ static void fil_name_process(const char *name, ulint len, uint32_t space_id,
 				break;
 			}
 
+			/* In case of prepare on partial backup,
+			InnoDB can't find some tablespace on target
+			directory. If InnoDB encounters redo log
+			for the tablespace before FILE_* record then
+			InnoDB should rename the file name in
+			recv_spaces from "" to name which encountered
+			during FILE_* record */
+			if (srv_prepare_partial_backup &&
+			    strcmp(fname.name.c_str(), "") == 0) {
```

Comment:
We don’t need `strcmp` to compare an empty string. Actually, the idiomatic C++ equivalent would be `fname.name == ""`. But, it should be more efficient to just check the length.

This function follows the old formatting style, where lines are split before, not after, binary operators.
```c++
			if (srv_prepare_partial_backup
			    && fname.name.size() == 0) {
```